relationships-api/src/main/java/org/commonjava/maven/atlas/graph/filter/OrFilter.java

```diff
@@ -58,6 +62,23 @@ public boolean accept( final ProjectRelationship<?, ?> rel )
     }
 
     @Override
+    public Set<ProjectRef> getDepExcludes()
+    {
+        Set<ProjectRef> excludes = new HashSet<ProjectRef>();
+        for (ProjectRelationshipFilter filter : getFilters())
+        {
+            Set<ProjectRef> filterExcludes = filter.getDepExcludes();
+            if (filterExcludes == null)
+            {
+                continue;
+            }
+
+            excludes.addAll( filterExcludes );
```

Comment:
This case is a bit worrisome. It's an 'or' filter, which implies that only one of the sub-filters need be applied...but here, we're aggregating the sub-filters' dependency excludes. I'm not sure what the right solution would be here, but this seems like it could introduce a bug (again, I'm not sure I could write that test yet).